s2spy/preprocess.py

```diff
@@ -61,9 +61,36 @@ def _subtract_trend(data: Union[xr.DataArray, xr.Dataset], method: str, trend: d
     raise NotImplementedError
 
 
-def _get_climatology(data: Union[xr.Dataset, xr.DataArray]):
+def _get_climatology(data: Union[xr.Dataset, xr.DataArray], timescale: str):
```

Comment:
You could also make timescale a literal: `Literal["monthly", "weekly", "daily", "hourly"]` ;)